### ISSUE TITLE
[BUG] Generated Chapters Are Jumbled Into Random Order

### ISSUE THREAD

**Issue by loriette-01** [maoserr/epublifier#188]
**Describe the bug**
Hello, here to report the second bug~ :D

After **retrieving the entire listing of chapters** from Novelupdates, I **parse the entire listing** to retrieve the direct titles from the translator's site, but then I **select only a specific range of chapters** that I want generated into the epub. But because I **parsed the entire listing of chapters** and **only selected the first few chapters** to pack into the epub, the chapters for some reason get **generated in a random mismatched order** in both the **html file themselves** and the **Table of Contents file itself** as well.

This **does not happen** if **ALL the parsed chapters are selected** to be generated into the epub. But if I **parse the entire listing of chapters** and then **only select a specific range of those parsed chapters** to pack into the epub, it will then be generated into a random jumbled order in the epub file (including in the Table of Contents file as well).

**Required info (please complete the following information):**
 - Url [e.g. specific series link]: Also happens for any novel link --> **https://www.novelupdates.com/series/after-failing-to-influence-the-protagonist/**
 - Browser [e.g. chrome, firefox]: **Chrome**
 - Extension Version [e.g. 2.0.24]: Latest Version --> **2.0.56**

**Comment by maoserr** [comment]
hmm, that's odd. Right now there's no specific sort order, so it should just generate based on how selected chapter are added. Are you selecting chapters randomly? If not, then this is a bug.

**Comment by vanessabluellie** [comment]
Ah no, I'm selecting chapters in the order that they are listed. I parsed the entire listing of chapters and then I selected only the beginning ten or so chapters. But because I parsed the entire listing of chapters previously, the selected range of chapters I chose to generate into the epub were all jumbled and in a different order. I assumed this was because I parsed EVERY chapter, but then only selected a chapter range to generate into the epub.
